src/main/java/domain/RandomTicketFactory.java

```diff
@@ -0,0 +1,23 @@
+package domain;
+
+import domain.numberscontainer.LottoNumber;
+import domain.numberscontainer.SixLottoNumbersDTO;
+import domain.numberscontainer.Ticket;
+
+import java.util.*;
+
+public class RandomTicketFactory {
+    private static final int FIRST_INDEX = 0;
+    private static final int SIXTH_INDEX = 6;
+
+    public static Ticket createTicket() {
+        return new Ticket(new SixLottoNumbersDTO(getShuffledList()));
```

Comment:
https://github.com/woowacourse/java-lotto/pull/126#discussion_r383013189 에 대한 답변 여기에 남겨 드립니다.

```suggestion
        return new Ticket(getShuffledList());
```

와 같이 생성할 수 있다면 말씀하신 복잡도를 줄일 수 있을 것 같습니다!